docs/deployment/ha/kubernetes_ha.md

```diff
@@ -52,6 +52,10 @@ In order to identify the Flink cluster, you have to specify a `kubernetes.cluste
 
   <pre>kubernetes.cluster-id: cluster1337</pre>
 
+- [kubernetes.namespace]({% link deployment/config.md %}#kubernetes-namespace) (required):
+The namespace that will be used for running the jobmanager and taskmanager pods.
+  <pre>kubernetes.namespace: cluster-namespace</pre>
```

Comment:
Why is this a required configuration? The default namespace should be `default`, right?

Reply:
this is not the required configuration but is an important configuration, refer to the [FLINK-20798] problem, if the user deployed FLINK cluster not on the default namespace then the deploy will fail because of the wrong namespace specified. so the document should remind of the user this configuration. and I will remote the required tag then submit.

Reply:
I think the problem is a different one as indicated by Yang Wang. We don't set the proper namespace when watching the ConfigMap. Hence, I would close this PR.